src/main/resources/db/changelog/changes/11-create-books-categories-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-categories-table
+      author: Bohdan20b
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: category_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
should these columns be a primary keys? I recommend you revise the many-to-many relations type, and how can db tables look like in that case,